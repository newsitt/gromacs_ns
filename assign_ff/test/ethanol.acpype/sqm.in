Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6     C        1.0330        0.0620        0.0730 
   6    C1        2.5470        0.0650        0.0790 
   8     O        3.0290       -0.0370        1.4110 
   1     H        0.6480        0.1300       -0.9490 
   1    H1        0.6410        0.9030        0.6540 
   1    H2        0.6470       -0.8530        0.5360 
   1    H3        2.9380        0.9850       -0.3660 
   1    H4        2.9340       -0.7850       -0.4900 
   1    H5        2.6890        0.7280        1.9050 

