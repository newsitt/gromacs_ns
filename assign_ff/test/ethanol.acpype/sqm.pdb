ATOM      1  C   UNL     1       1.027   0.075   0.075  1.00  0.00           C
ATOM      2  C1  UNL     1       2.538   0.081   0.088  1.00  0.00           C
ATOM      3  O   UNL     1       3.063  -0.051   1.396  1.00  0.00           O
ATOM      4  H   UNL     1       0.653   0.097  -0.976  1.00  0.00           H
ATOM      5  H1  UNL     1       0.625   0.967   0.615  1.00  0.00           H
ATOM      6  H2  UNL     1       0.638  -0.843   0.577  1.00  0.00           H
ATOM      7  H3  UNL     1       2.930   1.022  -0.385  1.00  0.00           H
ATOM      8  H4  UNL     1       2.950  -0.806  -0.466  1.00  0.00           H
ATOM      9  H5  UNL     1       2.682   0.656   1.930  1.00  0.00           H
