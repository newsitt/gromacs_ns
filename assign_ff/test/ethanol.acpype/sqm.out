            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 10

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       18
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.000458 seconds
| QMMM:                 Dspev diag routine = 0.000657 seconds
| QMMM:                Dspevd diag routine = 0.000641 seconds
| QMMM:                Dspevx diag routine = 0.002186 seconds
| QMMM:                 Dsyev diag routine = 0.000641 seconds
| QMMM:                Dsyevd diag routine = 0.000644 seconds
| QMMM:                Dsyevr diag routine = 0.000739 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000065 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.0330    0.0620    0.0730
  QMMM:     2        2      C        2.5470    0.0650    0.0790
  QMMM:     3        3      O        3.0290   -0.0370    1.4110
  QMMM:     4        4      H        0.6480    0.1300   -0.9490
  QMMM:     5        5      H        0.6410    0.9030    0.6540
  QMMM:     6        6      H        0.6470   -0.8530    0.5360
  QMMM:     7        7      H        2.9380    0.9850   -0.3660
  QMMM:     8        8      H        2.9340   -0.7850   -0.4900
  QMMM:     9        9      H        2.6890    0.7280    1.9050

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      -64.2694 kcal/mol        0.1052 kcal/(mol*A)
xmin    20      -64.2765 kcal/mol        0.0486 kcal/(mol*A)
xmin    30      -64.2781 kcal/mol        0.0459 kcal/(mol*A)
xmin    40      -64.2782 kcal/mol        0.0076 kcal/(mol*A)
xmin    50      -64.2782 kcal/mol        0.0072 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -38.2123  -31.3299  -23.6848  -17.9565  -15.9963  -15.4475  -13.6124  -12.9445
 -12.0055  -11.0486    3.6517    3.7654    4.1299    4.1778    4.5936    4.7572
   4.8924    6.3188

 Heat of formation   =        -64.27819265 kcal/mol  (       -2.78731159 eV)

 Total SCF energy    =     -15216.94429252 kcal/mol  (     -659.85622013 eV)
 Electronic energy   =     -41285.56379613 kcal/mol  (    -1790.27638854 eV)
 Core-core repulsion =      26068.61950361 kcal/mol  (     1130.42016841 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.254
     2      C                 -0.024
     3      O                 -0.327
     4      H                  0.084
     5      H                  0.073
     6      H                  0.088
     7      H                  0.061
     8      H                  0.104
     9      H                  0.195
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -1.168    1.010   -0.673    1.685
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.0266    0.0754    0.0752
  QMMM:     2        2      C        2.5379    0.0810    0.0876
  QMMM:     3        3      O        3.0627   -0.0513    1.3956
  QMMM:     4        4      H        0.6535    0.0969   -0.9760
  QMMM:     5        5      H        0.6255    0.9667    0.6150
  QMMM:     6        6      H        0.6377   -0.8428    0.5772
  QMMM:     7        7      H        2.9302    1.0222   -0.3849
  QMMM:     8        8      H        2.9495   -0.8058   -0.4663
  QMMM:     9        9      H        2.6823    0.6557    1.9297

           --------- Calculation Completed ----------

